### DISCUSSION TITLE
Help please: Can I style nodes simply based on my selection?

### DISCUSSION THREAD

**Discussion by KennyDenvers** [HEmile/juggl#104] [Q&A]
I am very new to Juggl. It looks impressive. One question I have is: Can I style nodes (or notes in the graph) simply based on the condition that I have selected them (with mouseclick or ctrl + mouseclick)? 
Or do I have to (manually) add a parameter (tag or yaml thing - new to that, too; and not familiar with classes etc.)? 

What would you recommend to use to create different groups or parameters for notes to be the basis for styles in Juggl?

What would you think would be most helpful to start out with to improve my styling options? (something like "classes" or whatever ...?)

Thanks for your help - I appreciate it very much!

**Comment by HEmile** [comment]
> Can I style nodes (or notes in the graph) simply based on the condition that I have selected them (with mouseclick or ctrl + mouseclick)?

Hmm, no. But that sounds interesting. You can add a feature request for this. With YAML you can definitely do this, but it's a bit of a hassle. Right here is some info on how to do this https://juggl.io/Features/Styling/YAML+Styling . The `cssclass` option in particular will allow you to assign classes to groups of nodes. 

Then you can use a `class:` filter to select those https://juggl.io/Features/Filtering
